README.md

```diff
@@ -15,51 +15,92 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(max(l))
 ```
-
 - [Min Value](https://repl.it/@Admin7/minvalue)
+
 ```python
-# copy & paste your solution here
+
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(min(l))
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+def return_extended(l, iter):
+  for i in iter:
+    l.append(i)
+  return l
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  result = []
+  for i in range(h):
+    row = []
+    for x in range(w):
+      row.append('#')
+    result.append(row)
+  return result
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
-# copy & paste your solution here
+def get_total(h, w):
+  total = 0
+  new = 0
+  for i in h:
+    for key,value in i.items():
+        if value == w:
+            total = i.get('subtotal')
```

Comment:
Close! Remember, we are summing up transactions here. `+=` would add AND reassign the `total` variable. whereas this just reassigns.